rainbow-me/rainbowkit
# feat: `@rainbow-me/rainbowkit/chains`

## Changes
- Added `@rainbow-me/rainbowkit/chains`

## What to test
1. Make sure you remove polygon, bsc, zkSync and polygonMumbai chains from your metamask wallet
2. Connect your wallet
3. Now try opening chain modal and try adding one of the chains mentioned in 1st point. You should see no warnings or errors when adding new chain to metamask

[Here](https://support.metamask.io/hc/en-us/articles/360057142392-Verifying-custom-network-information) is the page i've used to see what kind of valid info i needed for the chains. Metamask also uses [chainlist](https://chainlist.wtf/) info to do a "check" when adding a new chain.
